lib/rbac.ex

```diff
@@ -37,4 +37,117 @@ defmodule RBAC do
   def transform_role_list_to_string(roles) do
     [Map.delete(roles, :__meta__)] |> transform_role_list_to_string()
   end
+
+  @doc """
+  `get_approles/2` fetches the roles for the app
+  """
+  def get_approles(auth_url, client_id) do
+    url = "#{auth_url}/approles/#{client_id}"
+    HTTPoison.start()
```

Comment:
`mix` should start HTTPoison for us, we shouldn't have to call `HTTPoison.start/1` explicitly

Reply:
Yeah, agreed. For some reason it wasn't working when I was trying it the other day. 
If you have time, please rework it. (_you will need an `AUTH_API_KEY` to run it on `localhost`_)

Reply:
@th0mas Thanks for making this change. 👍